ui/v2.5/src/components/Performers/PerformerCard.tsx

```diff
@@ -61,17 +65,38 @@ export const PerformerCard: React.FC<IPerformerCardProps> = ({
     { age, years_old: ageL10String }
   );
 
-  function maybeRenderFavoriteIcon() {
-    if (performer.favorite === false) {
-      return;
-    }
-    return (
-      <div className="favorite">
+  const [updatePerformer] = usePerformerUpdate();
+
+  function maybeRenderFavoriteIcon(){
```

Comment:
The name of this method can be updated since the render is no longer conditional.

Reply:
totally agree, will update to just 'renderFavoriteIcon'